src/foam/u2/wizard/controllers/IncrementalWizardController.js

```diff
@@ -17,6 +17,10 @@ foam.CLASS({
     'foam.u2.wizard.axiom.WizardAction'
   ],
 
+  imports: [
+    'handleEvent?'
```

Comment:
Is this required id the parent class imports it as well?

Reply:
no need, removed